doc/nrf/versions.txt.in

```diff
@@ -71,6 +71,6 @@
 .. |recommonmark_ver| replace:: @RECOMMONMARK_VERSION@
 .. |sphinx_ver| replace:: @SPHINX_VERSION@
 .. |sphinx-tabs_ver| replace:: @SPHINX-TABS_VERSION@
-.. |sphinx-ncs-theme_ver| replace:: @SPHINX-NCS-THEME_VERSION@
+.. |sphinx-ncs-theme_ver| replace:: 0.6.4
```

Comment:
Change intentional?

Reply:
Yes..this was just to see the preview with the added style.. reverting this soon to point to the new release (the style changes have been merged).